src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <include resource="org/springframework/boot/logging/logback/base.xml"/>
+
+    <appender name="ALERT_AUDIT" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <encoder>
+			<pattern>${FILE_LOG_PATTERN}</pattern>
+		</encoder>
+		<file>log/alertAudit.log</file>
```

Comment:
Is the format strange here? I'm not sure if it's github or if it's actually offset

Reply:
I believe its Github, it looks normal in the IDE.